### PR TITLE
SOC-5929 Fix displayed username in navigation banner

### PR DESCRIPTION
**Pull request by boubaker**
This PR fixes the displayed name in banner when accessing navigation of other user's pages:
* activities
* wiki
* profile
* connections

In fact, the used regular expressions for URL patterns aren't generic, thus a new rule is added for those pages to include the case where we specify more than the username at the end of the URI.

### CODE REVIEW

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/common-configuration.xml [hunk 1]
```diff
@@ -52,14 +52,34 @@
...
+                    <key><string>activity.stream.owner.extended.show</string></key>
+                    <value><string><![CDATA[/activities/{streamOwnerId}/{<.*>path}]]></string></value>
+                  </entry>
                   <entry>
                     <key><string>activity.stream.owner.show</string></key>
                     <value><string>/activities/{streamOwnerId}</string></value>
                   </entry>
+                  <entry>
+                    <key><string>wiki.owner.extended.show</string></key>
+                    <value><string><![CDATA[/wiki/user/{streamOwnerId}/{<.*>path}]]></string></value>
```
Comment (thomasdelhomenie) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/common-configuration.xml]: Shouldn't we move these rules to wiki project instead ?

Comment (boubaker) [reply extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/common-configuration.xml]: you mean integration wiki ? good idea (y)

Comment (thomasdelhomenie) [reply extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/common-configuration.xml]: Why not wiki project ?

Comment (boubaker) [reply extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/common-configuration.xml]: We have in integration-wiki the listeners and configurations about integration of wiki into Social, thus this should be done in integration project IMO.

Comment (thomasdelhomenie) [reply extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/common-configuration.xml]: ok

Comment (boubaker) [reply extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/common-configuration.xml]: done with an extra ;-)

**Pull request merged**